.release-notes/4303.md

```diff
@@ -0,0 +1,6 @@
+## Enable building of libponyc-standalone on macos
+
+With this change we now produce the static library `libponyc-standalone.a` also on macos.
+As on osx it is hard to statically link `libc++`, it needs to be linked dynamically
+when building a final binary with this library.
```

Comment:
We now ship a "standalone" version of libponyc for MacOS. libponyc-standalone allows applications to use Pony compiler functionality as a library. The standalone version contains "all dependencies" needed in a single library. On MacOS, sadly "all dependencies" means "all that can be statically linked", so unlike Linux, dynamically linking to C++ standard library is required on MacOS.